src/py42/clients/_archive_access.py

```diff
@@ -31,17 +33,25 @@ def get_archive_accessor(
             device_guid, destination_guid=destination_guid
         )
         decryption_keys = self._get_decryption_keys(
-            device_guid, private_password, encryption_key
+            device_guid=device_guid,
```

Comment:
is there a reason all these keyword args got added? These aren't optional params.

Reply:
Because there are so many of them and trying to maintain the order is challenging. It became painful to work with, so I made it like this to ease the pain...  Is there a rule that key value args can't be required params? This is all internal stuff.

Reply:
I changed it back, but I still think positional args are harder to work with in these situations...